src/FSharpVSPowerTools.Logic/ProjectSystem/OpenDocumentsTracker.fs

```diff
@@ -15,23 +16,24 @@ type OpenDocument =
       Encoding: Encoding
       LastChangeTime: DateTime
       ViewCount: int }
-    member x.Text = lazy (x.Snapshot.GetText())
     static member Create document snapshot encoding lastChangeTime = 
         { Document = document
           Snapshot = snapshot
           Encoding = encoding
           LastChangeTime = lastChangeTime
           ViewCount = 1 }
+    member private x.text = lazy (x.Snapshot.GetText())
```

Comment:
Can't use Text without changing call sites due to how interfaces are explicitly implemented in F#, I'd need to change call sites and lose access to other properties.